### PR TITLE
build(lerna): evaluate lerna-lite migration

### LINKED ISSUE #1890: build(lerna): evaluate lerna-lite migration
Determine if we could migrate from upstream lerna to this fork given that
the upstream repo has not been receiving too much maintenance lately.

https://github.com/ghiscoding/lerna-lite

One upside for sure is that it would have the `--signoff` flag supported: https://github.com/lerna/lerna/pull/2897#issuecomment-1036641101

### PR DESCRIPTION
**Pull request by johnhomantaring**
Fixes #1890

Signed-off-by: johnhomantaring <john.h.o.mantaring@accenture.com>

### CODE REVIEW

**Comment by johnhomantaring** [comment]
Already doen with initial changes and lerna-lite is running properly:
image
image

**Comment by johnhomantaring** [comment]
--signoff is now supported as new packages has been created https://github.com/ghiscoding/lerna-lite..
image

I just removed lerna clean and lerna boostrap as they are not supported by lerna-lite.

**Review by petermetz — CHANGES_REQUESTED** [review]
@johnhomantaring Please fix the failing checks (build-node-16)

**Comment by johnhomantaring** [comment]
build-node-16 is failing due to "lerna list" as of last week this is not yet supported by lerna-lite. Already asked the dev if there is a roadmap that they would this as an option. Initially they've mentioned "No" but yesterday I was informed by the maintainer of lerna-lite that an optional feature for lerna list would be added. I will just monitor [PR](https://github.com/ghiscoding/lerna-lite/pull/171) once merged so I could continue testing lerna-lite and fix the failing build.

File: package.json
Comment (ghiscoding) [inline package.json:77]: hi there, Lerna-Lite maintainer here... you don't actually need to have `core`, neither `publish` since these 2 are already dependencies of `@lerna-lite/cli`, see Lerna-Lite [installation](https://github.com/ghiscoding/lerna-lite#installation) table. To be clear, you could also leave them there but it's just more dependencies that you'll have to keep updating, so up to you 😉 However `exec`, `list` and `run` must be kept because those are optional and not included in the CLI. Cheers and Thanks for trying Lerna-Lite

Comment (jagpreetsinghsasan) [reply package.json:77]: Thanks a lot @ghiscoding for the insights. :)

Comment (johnhomantaring) [reply package.json:77]: thank you @ghiscoding will try and test it!

**Comment by johnhomantaring** [comment]
Depends on #2096

**Comment by petermetz** [comment]
> Depends on #2096

@johnhomantaring You have to put the dependency declaration in the PR description. The bot won't scan the comments for it.

**Review by petermetz — APPROVED** [review]
@johnhomantaring LGTM, thank you!

**Review by petermetz — CHANGES_REQUESTED** [review]
@johnhomantaring Please resolve the merge conflicts and then we are good to go once again!

**Review by petermetz — APPROVED** [review]
@johnhomantaring LGTM, thank you!

**Pull request merged**
